scripts/parse_cd.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
```

Comment:
As we discussed, please incorporate your logic into the existing `parse_live.py` instead of creating a new script.